### DISCUSSION TITLE
[LARAVEL ITEGRATION]  - Tabler on laravel

### DISCUSSION THREAD

**Discussion by Nejcc** [tabler/tabler#1047] [Ideas]
9. February,  2022 - the new Laravel 9 release will came out. I wanna make this tabler template into laravel integration. 
I wanna ask you devs if there is any particular things that i need to follow, recommendation about new future release and updates?

- Laravel (PHP) framework
- SCSS
- js
- vuejs
- localization support
- more...

1. I notice that tabler use different includes for css / js  + scripts on the bottom of the *.html page (range picker, select, etc...)
    - is there any potions to migrate this in complete package or in particular order to work properly ?  
2. is there any purge css option to remove everything that is not in HTML page from css ? 
3. any other suggestions to follow ? 

@codecalm ?

Thank you very much

**Comment by kevinpapst** [comment]
I made a Symfony bundle, so if you need some inspiration: https://github.com/kevinpapst/TablerBundle

**Comment by Nejcc** [comment]
Sure i will check it ! Do you guys have any community hub like discord? @kevinpapst @codecalm ?

**Comment by Nejcc** [comment]
> I made a Symfony bundle, so if you need some inspiration: https://github.com/kevinpapst/TablerBundle

Do you have preview url?

**Comment by kevinpapst** [comment]
> Do you have preview url?

Only a demo app: https://github.com/kevinpapst/TablerBundle-Demo - but not hosted, you have to spin it up yourself.

**Comment by shukriYusof** [comment]
> 9. February,  2022 - the new Laravel 9 release will came out. I wanna make this tabler template into laravel integration.
>    I wanna ask you devs if there is any particular things that i need to follow, recommendation about new future release and updates?
> 
> * Laravel (PHP) framework
> * SCSS
> * js
> * vuejs
> * localization support
> * more...
> 
> 1. I notice that tabler use different includes for css / js  + scripts on the bottom of the *.html page (range picker, select, etc...)
>    
>    * is there any potions to migrate this in complete package or in particular order to work properly ?
> 2. is there any purge css option to remove everything that is not in HTML page from css ?
> 3. any other suggestions to follow ?
> 
> @codecalm ?
> 
> Thank you very much

i think you can compile it on webpack , sass or js. cause tabler also a part of npm packages.

**Comment by nejcdev** [comment]
> > 9. February,  2022 - the new Laravel 9 release will came out. I wanna make this tabler template into laravel integration.
> >    I wanna ask you devs if there is any particular things that i need to follow, recommendation about new future release and updates?
> > 
> > 
> > * Laravel (PHP) framework
> > * SCSS
> > * js
> > * vuejs
> > * localization support
> > * more...
> > 
> > 
> > 1. I notice that tabler use different includes for css / js  + scripts on the bottom of the *.html page (range picker, select, etc...)
> >    
> >    * is there any potions to migrate this in complete package or in particular order to work properly ?
> > 2. is there any purge css option to remove everything that is not in HTML page from css ?
> > 3. any other suggestions to follow ?
> > 
> > @codecalm ?
> > Thank you very much
> 
> i think you can compile it on webpack , sass or js. cause tabler also a part of npm packages.

yes i will do that  it's already in progress?

**Comment by aliqasemzadeh** [comment]
https://github.com/alighasemzadeh/bap
You can use it but Laravel 8.
I will update it soon.
Livewire + Tabler.

**Comment by takielias** [comment]
This is the Latest Laravel Admin Dashboard Scaffolding Package with built-in Auth https://github.com/takielias/tablar. You may give it a try.
